src/Standards/Generic/Tests/NamingConventions/AbstractClassNamePrefixUnitTest.1.inc

```diff
@@ -56,4 +56,10 @@ class NameAbstractBar {}
 abstract class abstractOkName
 {
 
-}
\ No newline at end of file
+}
+
+final class FinalClassShouldNotTriggerWarning {}
```

Comment:
What about also adding a test with a `readonly` class ? and an `abstract readonly` class ?

Reply:
Done